conda/install.py

```diff
@@ -139,28 +139,27 @@ def win_conda_bat_redirect(src, dst, shell):
             else:
                 raise
 
-        if 'cmd.exe' in shell.lower():
-            # bat file redirect
-            with open(dst+'.bat', 'w') as f:
-                f.write('@echo off\n"%s" %%*\n' % src)
-
-        elif 'powershell' in shell.lower():
-            # TODO: probably need one here for powershell at some point
-            pass
-
-        else:
-            # This one is for bash/cygwin/msys
-            with open(dst, "w") as f:
-                f.write("#!/usr/bin/env bash \n")
-                if src.endswith("conda"):
-                    f.write('%s "$@"' % shells[shell]['path_to'](src+".exe"))
-                else:
-                    f.write('source %s "$@"' % shells[shell]['path_to'](src))
-            # Make the new file executable
-            # http://stackoverflow.com/a/30463972/1170370
-            mode = os.stat(dst).st_mode
-            mode |= (mode & 292) >> 2    # copy R bits to X
-            os.chmod(dst, mode)
+        # bat file redirect
```

Comment:
The docstring here says this function is for WindowsXP.  I guess that's not the case anymore?